tests/disas/pcc-imported-memory.wat

```diff
@@ -37,22 +37,22 @@
 ;; wasm[0]::function[0]:
 ;;       pushq   %rbp
 ;;       movq    %rsp, %rbp
-;;       movq    0x38(%rdi), %rcx
-;;       movq    8(%rcx), %rax
-;;       shrq    $0x10, %rax
-;;       movq    %rax, %rcx
-;;       shll    $0x10, %ecx
-;;       leal    4(%rax), %edx
-;;       cmpl    %edx, %ecx
-;;       jbe     0x3b
-;;   21: testl   %eax, %eax
-;;       jle     0x3b
-;;   29: movq    0x38(%rdi), %rsi
-;;       movq    (%rsi), %rsi
-;;       movl    %eax, %edi
-;;       movl    (%rsi, %rdi), %r10d
-;;       jmp     0x3e
-;;   3b: xorl    %r10d, %r10d
+;;       movq    0x38(%rdi), %rdx
+;;       movq    8(%rdx), %rcx
+;;       shrq    $0x10, %rcx
+;;       movq    %rcx, %rdx
+;;       shll    $0x10, %edx
+;;       leal    4(%rcx), %r8d
+;;       cmpl    %r8d, %edx
+;;       jbe     0x3d
+;;   23: testl   %ecx, %ecx
```

Comment:
...whereas here we've managed to gain two of them! It's not yet clear to me why this is.